src/Bootstrap/src/Autoconfig/Steeltoe.Bootstrap.Autoconfig.csproj

```diff
@@ -13,30 +13,29 @@
-    <ProjectReference Include="..\..\..\Configuration\src\PlaceholderCore\Steeltoe.Extensions.Configuration.PlaceholderCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\CloudFoundry\Steeltoe.Extensions.Configuration.CloudFoundry.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\ConfigServer\Steeltoe.Extensions.Configuration.ConfigServer.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\Kubernetes\Steeltoe.Extensions.Configuration.Kubernetes.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\RandomValue\Steeltoe.Extensions.Configuration.RandomValue.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Configuration\src\Placeholder\Steeltoe.Extensions.Configuration.Placeholder.csproj" PrivateAssets="All" />
     <ProjectReference Include="..\..\..\Connectors\src\Connector.EF6Core\Steeltoe.Connector.EF6Core.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Connectors\src\ConnectorCore\Steeltoe.Connector.ConnectorCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Discovery\src\ClientCore\Steeltoe.Discovery.ClientCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Logging\src\DynamicSerilogCore\Steeltoe.Extensions.Logging.DynamicSerilogCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Management\src\CloudFoundryCore\Steeltoe.Management.CloudFoundryCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Management\src\EndpointCore\Steeltoe.Management.EndpointCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Management\src\KubernetesCore\Steeltoe.Management.KubernetesCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Management\src\TaskCore\Steeltoe.Management.TaskCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Management\src\TracingCore\Steeltoe.Management.TracingCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Security\src\Authentication.CloudFoundryCore\Steeltoe.Security.Authentication.CloudFoundryCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Security\src\Authentication.MtlsCore\Steeltoe.Security.Authentication.MtlsCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Security\src\DataProtection.CredHubCore\Steeltoe.Security.DataProtection.CredHubCore.csproj" PrivateAssets="All" />
-    <ProjectReference Include="..\..\..\Security\src\DataProtection.RedisCore\Steeltoe.Security.DataProtection.RedisCore.csproj" PrivateAssets="All" />
+    <ProjectReference Include="..\..\..\Connectors\src\Connector\Steeltoe.Connector.Connector.csproj" PrivateAssets="All" />
```

Comment:
Should this just be `Steeltoe.Connector`? Same goes for `Steeltoe.Integration`, `Steeltoe.Messaging`, `Steeltoe.Stream` and corresponding test projects

Reply:
My preference is to remove this redundancy, so I changed it in 436eda973edbdec35ee3d0e46f30ec69d0f32b6f.... I'm still open to discussion though

Reply:
I misread this to mean something else. Now that I've seen your changes I'm fully onboard with them.